metals/src/main/scala/scala/meta/internal/metals/codeactions/MillifyScalaCliDependencyCodeAction.scala

```diff
@@ -0,0 +1,113 @@
+package scala.meta.internal.metals.codeactions
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+
+import scala.meta.internal.metals.Buffers
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.internal.metals.codeactions.CodeAction
+import scala.meta.internal.metals.codeactions.MillifyScalaCliDependencyCodeAction._
+import scala.meta.internal.parsing.Trees
+import scala.meta.io.AbsolutePath
+import scala.meta.pc.CancelToken
+import scala.meta.tokens.Token._
+
+import org.eclipse.{lsp4j => l}
+
+// TODO prepare LSP spec if it works in manual tests
+// there are some similarities to MillifyDependencyCodeAction so maybe they can be extracted
+class MillifyScalaCliDependencyCodeAction(buffers: Buffers) extends CodeAction {
```

Comment:
You can take a look at `ScalaCliActionsSuite` as an example

Reply:
Sure thing, tests are now implemented